lib/api/element-commands/_waitFor.js

```diff
@@ -1,6 +1,8 @@
 const Utils = require('../../utils');
 const {AssertionRunner} = require('../../assertion');
 const ElementCommand = require('../../element').Command;
```

Comment:
you could more elegantly write the 2 lines as:
`const {LocateStrategy, Command: ElementCommand} = require('../../element');`

Reply:
done ✅